pkgs/by-name/ta/tachidesk-server/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, makeWrapper
+, jdk17_headless
+, nixosTests
+}:
+
+let
+  jdk = jdk17_headless;
+in
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "tachidesk-server";
+  version = "0.7.0";
+  revision = 1197;
+
+  src = fetchurl {
+    url = "https://github.com/Suwayomi/Tachidesk-Server/releases/download/v${finalAttrs.version}/Tachidesk-Server-v${finalAttrs.version}-r${toString finalAttrs.revision}.jar";
```

Comment:
By the way, upstream seems to have changed their name of the project to `Suwayomi-Server`. It might be easier to rename it now than later.

Reply:
Indeed, it was quite complicated: I discussed this with the maintainers, and they me that I should keep tachidesk-server at least until the next complete release. However, because of the recent events that happened to Tachiyomi (the android app, that basically got down because of copyright issue), the name has changed quicker than I thought it would.

Reply:
I change the name tachidesk to suwayomi everywhere but the executable main that hasn't changed since no new release has been published

Reply:
That makes sense. RIP tachiyomi